### PR TITLE
fix: add assertion in unit test of OpenmldbCatalogService

### LINKED ISSUE #1317: Add assertion in unit test of OpenmldbCatalogService
Now the unit test of `OpenmldbCatalogService` is quite simple and lack of assertions. You can find the test code in https://github.com/4paradigm/OpenMLDB/blob/main/java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/catalog/TestOpenmldbCatalogService.scala .

Now we can get all the table names from internal db. It would be better to add code in line 36 and check the table name like this using assertation.
```
val internalTableNames = List("JOB_INFO", "PRE_AGG_META_INFO")
assert(internalTableNames.contains(tables(i)))
```

### PR DESCRIPTION
**Pull request by SaumyaBhushan**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
* Test

* **What is the current behavior?** (You can also link to an open issue here)
* https://github.com/4paradigm/OpenMLDB/issues/1317
close #1317 

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

File: java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/catalog/TestOpenmldbCatalogService.scala [hunk 1]
```diff
@@ -33,6 +33,8 @@ class TestOpenmldbCatalogService extends FunSuite {
       println(s"Database $i: ${databases(i)}")
     }
 
+    val internalTableNames = List("JOB_INFO", "PRE_AGG_META_INFO")
+    assert(internalTableNames.contains(tables(i)))
```
Comment (tobegit3hub) [inline java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/catalog/TestOpenmldbCatalogService.scala]: You need to get `tables(i)` in the following `for` loop.

**Comment by tobegit3hub** [comment]
@SaumyaBhushan It would be great if you can change the code for my comment.

**Comment by SaumyaBhushan** [comment]
> @SaumyaBhushan It would be great if you can change the code for my comment.

Sure @tobegit3hub .

File: java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/catalog/TestOpenmldbCatalogService.scala (hunk 1 above)
Comment (SaumyaBhushan) [reply java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/catalog/TestOpenmldbCatalogService.scala]: I am not sure, In scala how to do that.

Comment (SaumyaBhushan) [reply java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/catalog/TestOpenmldbCatalogService.scala:37]: I am not sure, In scala how to do that.

Comment (tobegit3hub) [reply java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/catalog/TestOpenmldbCatalogService.scala]: Jost move these added code in line 40.

**Review by vagetablechicken — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Pull request merged**